content/links/3ba7d4d792a40c1eec4065dba304e22201b0663f0b3051b781ceff23b4c90a82.md

```diff
@@ -0,0 +1,12 @@
+---
+source_url: "https://stratechery.com/2020/how-tech-can-build/"
+title: "How Tech Can Build"
+author: "Ben Thompson"
+submitter:
+    name: "Og Maciel"
+    url: "https://twitter.com/OgMaciel"
```

Comment:
Hmm... Do we want to do that for every link, or only for these submitted by "external" people?

I see how one of links I have added could make you think that I want to add this for all the links, but that was there only for me to test layout :) .

Reply:
Hmmm, in that case, we can keep it as you originally intended.

Reply:
I wouldn't mind at all. Had to rebase, commited in c052c501ea068c560909aec06b3bb9bc320d18cf.